eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -39,7 +39,9 @@ stages:
       - stage: Release_${{artifact.safeName}}
         displayName: 'Release: ${{artifact.name}}'
         dependsOn: Signing
-        condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net-pr'))
+        condition: >- # 1191 is the system.definitionId for 'net - template' pipeline
```

Comment:
I'm not a fan of using the definitionId in this way. I'd prefer another variable that we could just set in the template project.

@mitchdenny what is your thoughts here? We want to enable automatic triggering of the internal template pipeline for team members so that we can more easily test eng/common changes all the way through release.

Reply:
Yeah, I don't think that this is a good idea (too fragile). If we have to regenerate the pipeline for whatever reason this will break. I'm wondering why we would want to opt this pipeline out of signing anyway?

Reply:
A few things we need to consider here also. Are we opening a way to execute code on pipelines where there is access to credentials without sufficient safeguards?